host/hackrf-tools/src/hackrf_sweep.c

```diff
@@ -349,22 +353,23 @@ static void usage() {
 	fprintf(stderr, "\t[-p antenna_enable] # Antenna port power, 1=Enable, 0=Disable\n");
 	fprintf(stderr, "\t[-l gain_db] # RX LNA (IF) gain, 0-40dB, 8dB steps\n");
 	fprintf(stderr, "\t[-g gain_db] # RX VGA (baseband) gain, 0-62dB, 2dB steps\n");
-	fprintf(stderr, "\t[-w bin_width] # FFT bin width (frequency resolution) in Hz, 2445-5000000\n");
+	fprintf(stderr,
```

Comment:
Hmm.
Maybe these long printout blocks should be handled as one long string anyway.

Reply:
Maybe, but there are several parameters interpolated into that one big string, and it's much easier to see where each ones fit in the format string when it's done line by line.

The best way to improve this section might be to add an `err()` macro rather than writing out `fprintf(stderr, ...)` every time.

Reply:
I've replaced this whole block with a single call to `fprintf`, using string literal concatenation for the lines.